front/cypress/integration/sign_up_email_password.ts

```diff
@@ -121,6 +118,8 @@ describe('Sign up - Email + password step', () => {
 
   it('signs up successfully', () => {
     signUp();
+    cy.get('.e2e-modal-close-button').first().click();
+    cy.logout();
```

Comment:
Why is this needed? Doesn't every test start with a clean sheet?

Reply:
Apparently not, without this the next test starts with the user still being logged in